serialzy/serializers/base_model.py

```diff
@@ -69,13 +69,45 @@ def deserialize_from_dir(source: BinaryIO, read_from_dir) -> Any:
         return model
 
 
+def unpack_model_file(source: BinaryIO, destination: Path) -> None:
+    with destination.open("wb") as handle:
+        while True:
+            data = source.read(8096)
+            if not data:
+                break
+            handle.write(data)
+        handle.flush()
+        os.fsync(handle.fileno())
+
+
+def unpack_model_tar(source: BinaryIO, destination: Path) -> None:
+    with tempfile.NamedTemporaryFile(suffix=".tar") as handle:
+        while True:
+            data = source.read(8096)
+            if not data:
+                break
+            handle.write(data)
+        handle.flush()
+        os.fsync(handle.fileno())
+        shutil.unpack_archive(handle.name, destination, "tar")
+
+
 class ModelBaseSerializer(DefaultSchemaSerializerByReference, ABC):
     META_FILE_NAME = 'meta.json'
 
     def __init__(self, module: str, serializer_name: str):
         self.module = module
         self.logger = logging.getLogger(serializer_name)
 
+    @abstractmethod
+    def unpack_model(self, source: BinaryIO, dest_dir: str) -> None:
```

Comment:
I don't like the fact that the user can't know where exactly will be extracted model.
Why are you getting dest_dir instead of dest_filename, for example?
And if it is okay, maybe you should return model filename or path from this method?

Reply:
модель будет в dest_dir, что за данные там будут знает только сериализатор

Reply:
But somebody will open this model later?
How it will know model filename?